README.md

```diff
@@ -451,39 +444,20 @@ See `docs/CHANGELOG.md`.
 
 ## Background
 
-### Why was this fork created?
+### What's the plan and why?
 
-The original Bats repository needed new maintainers, and has not been actively
-maintained since 2013. While there were volunteers for maintainers, attempts to
-organize issues, and outstanding PRs, the lack of write-access to the repo
-hindered progress severely.
+**Tuesday, September 19, 2017:** This was forked from [Bats][bats-orig] at
+commit [0360811][].  It was created via `git clone --bare` and `git push
+--mirror`. See the [Background](#background) section above for more information.
 
-### What's the plan and why?
+[bats-orig]: https://github.com/sstephenson/bats
+[0360811]: https://github.com/sstephenson/bats/commit/03608115df2071fff4eaaff1605768c275e5f81f
+
+This [bats-core repo](https://github.com/bats-core/bats-core) is now the official Bats project.
```

Comment:
I'm not sure anyone is in the position of claiming which project is "official" other than Sam himself. I think we should reword this to indicate it's the main community project or something?